source/content/plugins-known-issues.md

```diff
@@ -851,7 +851,20 @@ the `advanced-cache.php` file.
 
 **Issue 2:** WP-rocket [assumes write access](/symlinks-assumed-write-access) to read-only file paths in Pantheon.
 
-**Solution:** [Create symlinks](#assumed-write-access) as noted above.
+**Solution 1:** [Specify a Custom Cache Folder and Config Path](https://docs.wp-rocket.me/article/1118-specify-a-custom-cache-folder) by adding this in the wp-config.php:
+
+Only works with WP Rocket 3.2 and newer:
+```
+define( 'WP_ROCKET_CACHE_ROOT_PATH', $_SERVER['DOCUMENT_ROOT'] . 'wp-content/uploads/new-path/cache/' ); 
+```
+
+Only works with WP Rocket 3.5 and newer:
+```
+define( 'WP_ROCKET_CONFIG_PATH', $_SERVER['DOCUMENT_ROOT'] . 'wp-content/uploads/wp-rocket-config/' );
+```
+
+
+**Solution 2:** [Create symlinks](#assumed-write-access) as noted above.
```

Comment:
@carl-alberto Is this section saying that from 3.2-3.4 you can define the `wp_rocket_cache_root_path` but would still need a symlink for the config path?

Reply:
Yes @alexfornuto, maybe add that note on that section please

May I also request this PR to be tagged as `hacktoberfest-accepted` please? :)

Reply:
@carl-alberto please take a look at the current version and confirm that it's still accurate, and better clarified the options per version.